tests/e2e/fixtures.js

```
@@ -128,22 +143,95 @@ class AIFeedbackUtils {
+		await use(new PageUtils({ page }));
 	},
 	editor: async ({ page }, use) => {
 		await use(new Editor({ page }));
 	},
-	pageUtils: async ({ page }, use) => {
-		await use(new PageUtils({ page }));
+	admin: async ({ page, pageUtils, editor }, use) => {
+		await use(new Admin({ page, pageUtils, editor }));
 	},
-	requestUtils: async ({}, use) => {
+	requestUtils: async ({ }, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)